streams/test-utils/src/test/java/org/apache/kafka/streams/TopologyTestDriverTest.java

```diff
@@ -305,40 +168,36 @@ private Topology setupTopologyWithTwoSubtopologies() {
 
 
     private Topology setupSingleProcessorTopology() {
-        return setupSingleProcessorTopology(-1, null, null);
+        return setupSingleProcessorTopology(-1, null);
     }
 
     private Topology setupSingleProcessorTopology(final long punctuationIntervalMs,
-                                                  final PunctuationType punctuationType,
-                                                  final Punctuator callback) {
```

Comment:
`MockProcessor` does not support callbacks...

Reply:
Similar as above. The only callback that was used is the MockPunctuator which counts the `punctuateAt` timestamps, this is already covered in current `MockProcessor`. My philosophy is that, until some feature was really required in the added test case, we can just keep those mocks as simple as possible.